core/include/vecmem/utils/types.hpp

```diff
@@ -6,13 +6,6 @@
  */
 #pragma once
 
-// CUDA include(s).
-#ifdef __CUDACC__
-#   include <cuda_runtime.h>
-#else
-typedef int cudaError_t; ///< Dummy type for @c cudaError_t in C++ code
-#endif // __CUDACC__
-
```

Comment:
I had to rip this out temporarily because it was causing a ton of duplicate definition errors. Of course we should put it back in when we're in a more stable position (if it is still necessary)!